src/test/java/services/tests/GAAccountTest.java

```diff
@@ -0,0 +1,39 @@
+package services.tests;
```

Comment:
Lets put all the services tests inside : com.training.services.tests
Also can you refactor all web tests to : com.training.web.tests?